config/config.go

```diff
@@ -219,6 +238,59 @@ func (c *Config) Validate() error {
 	c.sourceSubnetIDs = sourceSubnetIDs
 	c.sourceBlockchainIDs = sourceBlockchainIDs
 
+	// Validate the manual warp messages
+	for i, msg := range c.ManualWarpMessages {
+		if err := msg.Validate(); err != nil {
+			return fmt.Errorf("invalid manual warp message at index %d: %v", i, err)
+		}
+		c.ManualWarpMessages[i] = msg
+	}
+
+	return nil
+}
+
+func (m *ManualWarpMessage) GetUnsignedMessageBytes() []byte {
+	return m.unsignedMessageBytes
+}
+func (m *ManualWarpMessage) GetSourceBlockchainID() ids.ID {
+	return m.sourceBlockchainID
+}
+func (m *ManualWarpMessage) GetSourceAddress() common.Address {
+	return m.sourceAddress
+}
+func (m *ManualWarpMessage) GetDestinationBlockchainID() ids.ID {
+	return m.destinationBlockchainID
+}
+func (m *ManualWarpMessage) GetDestinationAddress() common.Address {
+	return m.destinationAddress
+}
+
+func (m *ManualWarpMessage) Validate() error {
```

Comment:
We may want to consider adding a comment documenting that `Validate` initializes the private fields of the struct.